src/components/Weather.tsx

```diff
@@ -38,21 +40,21 @@ const Weather: React.FC<ComponentProps> = ({ weather, setCityName }) => {
           </Grid>
           <Grid item>
             <Button
+              type="submit" // type "submit" will make this button as primary submit on "Enter" keyboard event
```

Comment:
type "submit" is used for the form submit Event. In other word:
_If a button inside a form has type "submit", it will call the onSubmit event of <form> in line 26 when the user clicks the button or presses `enter` key on any children input field in the form._